sigar_linux.go

```diff
@@ -720,6 +723,35 @@ func (self *ProcTime) Get(pid int) error {
 	self.StartTime += system.btime
 	self.StartTime *= 1000
 
+	return err
+}
+
+// Diff the counters to calculate percent of CPU usage. The "other" ProcTime should be from an earlier reading.
+func (self *ProcTime) Diff(other *ProcTime) error {
```

Comment:
nit: Thinking about it, I'm not sure the name "Diff" explains _why_ someone would call this method.  Maybe something like `CalculatePercentages` makes more sense?